python/llm/src/bigdl/llm/transformers/model.py

```diff
@@ -333,7 +333,7 @@ def from_gguf(fpath: str, optimize_model: bool = True, cpu_embedding: bool = Fal
         from bigdl.llm.optimize import optimize_model as optimize_model_fn
 
         model, tokenizer, low_bit = load_gguf_model(fpath, dtype=torch.half)
-        model = optimize_model_fn(model, low_bit=low_bit, optimize_llm=optimize_model,
+        model = optimize_model_fn(model, low_bit="sym_int4", optimize_llm=optimize_model,
```

Comment:
why always "sym_int4"?

Reply:
Refer to [here](https://github.com/intel-analytics/BigDL/blob/main/python/llm/src/bigdl/llm/transformers/gguf/models/llama.py#L81), each gguf model is always loaded as sym_int4 defaultly.